paypro-gateways-woocommerce/includes/paypro/wc/plugin.php

```diff
@@ -4,7 +4,7 @@ class PayPro_WC_Plugin
 {
     const PLUGIN_ID = 'paypro-gateways-woocommerce';
     const PLUGIN_TITLE = 'PayPro Gateways - WooCommerce';
-    const PLUGIN_VERSION = '1.3.0';
+    const PLUGIN_VERSION = '1.3.1';
```

Comment:
Plugin moet al naar versie `1.3.2`, tijdens de release de versie al naar `1.3.1` gezet. Dit is alleen niet in de code aangepast.